airflow/providers/google/cloud/operators/dataflow.py

```diff
@@ -945,6 +948,11 @@ def on_kill(self) -> None:
             )
 
 
+@deprecated(
+    reason="DataflowStartSqlJobOperator is deprecated and will be removed after 31.01.2025. "
+    "Please use DataflowStartYamlJobOperator instead.",
+    category=AirflowProviderDeprecationWarning,
+)
```

Comment:
Please check that it complies with the format that you suggested in https://github.com/apache/airflow/pull/41637 :)

Not a blocker for merging the current PR, just to reduce future work.

Reply:
Good point, thank you! The deprecation is updated.